app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: ENV["FROM_EMAIL"]
```

Comment:
Can you make a todo in this PR that we also add this ENV var to intercity/intercity-docker?